cpp/tests/utilities_tests/column_utilities_tests.cpp

```diff
@@ -559,8 +566,9 @@ TEST_F(ColumnUtilitiesStructsTest, Values)
 
   // equivalent, but not equal
   CUDF_TEST_EXPECT_COLUMNS_EQUIVALENT(s_col0, s_col1);
-  EXPECT_EQ(cudf::test::expect_columns_equal(s_col0, s_col1, cudf::test::debug_output_level::QUIET),
-            false);
+  EXPECT_EQ(
+    cudf::test::detail::expect_columns_equal(s_col0, s_col1, cudf::test::debug_output_level::QUIET),
+    false);
```

Comment:
Instead of "expect equal to false", just use "expect false". This applies in lots of places in this file.
```suggestion
  EXPECT_FALSE(cudf::test::detail::expect_columns_equal(s_col0, s_col1, cudf::test::debug_output_level::QUIET));
```